po/hu.po

```diff
@@ -20,132 +20,133 @@ msgstr ""
 #: data/com.github.tchx84.Flatseal.desktop.in:3
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:3 src/aboutDialog.ui:14
 msgid "Flatseal"
-msgstr ""
+msgstr "Flatseal"
 
 #: data/com.github.tchx84.Flatseal.desktop.in:9
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:8
 msgid "A permissions manager for Flatpak"
-msgstr ""
+msgstr "A Flatpak engedélykezelője"
 
 #: data/com.github.tchx84.Flatseal.desktop.in:10
 msgid "com.github.tchx84.Flatseal"
-msgstr ""
+msgstr "com.github.tchx84.Flatseal"
 
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:10 src/aboutDialog.ui:12
 msgid ""
 "Flatseal is a graphical utility to review and modify basic permissions from "
 "your Flatpak applications."
-msgstr ""
+msgstr "A Flatseal egy grafikus segédprogram, amely áttekintheti és "
+"módosíthatja a Flatpak-alkalmazások alapvető engedélyeit."
 
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:29
 msgid "Martin Abente Lahaye"
-msgstr ""
+msgstr "Lahaye Martin Abente"
 
 #: src/aboutDialog.ui:11
 msgid "Copyright © 2020 - Martín Abente Lahaye"
-msgstr ""
+msgstr "Szerzői jog © 2020 - Lahaye Martín Abente"
 
 #: src/applicationRow.ui:57
 msgid "Application"
-msgstr ""
+msgstr "Alkalmazás"
 
 #: src/applicationRow.ui:74 src/permissionEntryRow.ui:45
 #: src/permissionSwitchRow.ui:45
 msgid "Description"
-msgstr ""
+msgstr "Leírás"
 
 #: src/permissionEntryRow.ui:29 src/permissionSwitchRow.ui:29
 msgid "Pemission"
-msgstr ""
+msgstr "Engedély"
 
 #: src/view.ui:36
 msgid "No applications found."
-msgstr ""
+msgstr "Alkalmazás nem található."
 
 #: src/window.ui:41
 msgid "Reset"
-msgstr ""
+msgstr "Visszaállítás"
 
 #: src/window.ui:47
 msgid "Reset this application permisions"
-msgstr ""
+msgstr "Alkalmazás engedélyek alaphelyzetbe állítása"
 
 #: src/model.js:35
 msgid "Access network"
-msgstr ""
+msgstr "A hálózat hozzáférése"
 
 #: src/model.js:40
 msgid "Access inter-process communications"
-msgstr ""
+msgstr "A folyamatok közötti kommunikáció hozzáférése"
 
 #: src/model.js:45
 msgid "Access X11 windowing system"
-msgstr ""
+msgstr "Az X11 ablakrendszer hozzáférése"
 
 #: src/model.js:50
 msgid "Access X11 windowing system (as fallback)"
-msgstr ""
+msgstr "Az X11 ablakrendszer hozzáférése (tartalékként)"
 
 #: src/model.js:55
 msgid "Access Wayland windowing system"
-msgstr ""
+msgstr "A Wayland ablakrendszer hozzáférése"
 
 #: src/model.js:60
 msgid "Access PulseAudio sound server"
-msgstr ""
+msgstr "A PulseAudio hangkiszolgáló hozzáférése"
 
 #: src/model.js:65
 msgid "Access D-Bus system bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus rendszer busz hozzáférése (korlátlan)"
 
 #: src/model.js:70
 msgid "Access D-Bus session bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus rendszer busz hozzáférése (korlátlan)"
```

Comment:
Can you make it clear that this permission is for D-Bus session bus?

Reply:
Done